### DISCUSSION TITLE
Sandboxie not clean up after browser closes

### DISCUSSION THREAD

**Discussion by BilleBarrett** [sandboxie-plus/Sandboxie#3636] [General]
Whenever I open up our chrome Microsoft edge or Firefox and then I terminate them they still remain in the view of the sandboxy when I say terminate all attaches a picture
![1A-Sandboxie](https://github.com/sandboxie-plus/Sandboxie/assets/41338917/4fe3d260-bdb9-4754-ac83-57d56e58c77d)

**Comment by isaak654** [comment]
Go to the `View` menu -> disable the `Keep terminated` entry.

**Comment by BilleBarrett** [comment]
Not sure ho0w that get set 

 

-----------------------------------------------------------------------------------------------------------------


The bridge and the wall are made with the same material

The Bridge joins the people and the Wall divides the people

Choose the Right One

Remote Desktop Services 
(336) 988-0891    
 ***@***.***> ***@***.***

**Reply by isaak654** [reply]
You may have accidentally pressed [this button](https://raw.githubusercontent.com/sandboxie-plus/Sandboxie/20390f426368fc0c754aab63296ee9736732dddc/SandboxiePlus/SandMan/Resources/Actions/keep.png) located in the toolbar, which does the same thing.
